src/component/mxgraph/Options.ts

```diff
@@ -0,0 +1,19 @@
+/**
+ * Copyright 2020 Bonitasoft S.A.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+export enum FitType {
+  Default,
```

Comment:
:question: I would prefer `None` because `Default` does not explicit what is the default value.

Reply:
No problem

Reply:
I don't understand what `the default value is not accessible by a dedicated enum entry` means ?
If you talk about the demo page; there is an option to select this mode:
`<option value="Default" selected="yes">Default</option>`

Reply:
I meant that there is no value with an explicit name in the enum that matches the behaviour when there is no fit. Default does not explain what is the behaviour in that case.
Anyway, you already renamed it :smile: